security/s2a/internal/conn/expander.go

```diff
@@ -0,0 +1,10 @@
+package conn
+
+import "hash"
+
+// HKDFExpander is the interface for an HMAC-based Extract-and-Expand Key
+// Derivation Function (HKDF) expander. This is specified in Section 7 of
+// https://tools.ietf.org/html/rfc8446
+type HKDFExpander interface {
+	ExpandLabel(hash func() hash.Hash, secret, label []byte, length int) ([]byte, error)
```

Comment:
Can we call the method `Expand`?

Reply:
I think that may lead to a bit of confusion since the RFC makes a distinction between Expand and Expand Label. WDYT?

Reply:
The "HKDF-Expand-Label" terminology is, as far as I'm aware, local to the TLS 1.3 RFC. Indeed, they define it as the start of section 7 in the terms of "HKDF-Expand". The latter function, "HKDF-Expand", is the more commonly used and known (and it even has its own RFC). For that reason, I think it's better to use "HKDF-Expand". WDYT?

Reply:
Changed as per discussion.